wp-content/themes/cambodia-ict-camp/i18n/km_KH.po

```diff
@@ -26,32 +26,28 @@ msgid "Event Booking"
 msgstr ""
 
 #: hooks/slider-selection.php:195
-#, fuzzy
-#| msgid "The First Day of Cambodia ICT Camp is opening now"
 msgid "The First Day of Cambodia ICT Camp is opening now."
-msgstr "ថ្ងៃទី ១ នៃកម្មវិធីកម្ពុជាអាយស៊ីធីខែមចាប់ដំណើរការហើយ"
+msgstr "ថ្ងៃទី ១ នៃកម្មវិធី កម្ពុជា អាយស៊ីធីខែម កំពុងដំណើរការ"
```

Comment:
Please check with the editor of the format of using space in Khmer sentences

Reply:
Editor Team prefers the one without space. Therefore, I will make the changes accordingly.